src/ripple/app/misc/impl/Manifest.cpp

```diff
@@ -288,6 +288,19 @@ ManifestCache::applyManifest (Manifest m)
 
     bool const revoked = m.revoked();
 
+    if (revoked)
```

Comment:
Not opposed to this reordering, but the messages will now be slightly more confusing: first an "revoked" then an "accepted".

No need to change anything; just pointing it out.

Reply:
I didn't realize that.  But I wasn't 100% positive that the reordering wouldn't have side effects, so am grateful for reviewers to point out issues like this.

We could also copy instead of move, or chase down the moved-to location.

Reply:
It's not a big deal. This fix is sufficient for now, and I'm working on this code in another PR and will restructure some of this.